bin/bootstrap-dev.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+"""This script helps bootstrap users and data"""
+
+import os
+import os.path
+import sys
+import json
+import logging
+import argparse
+import datetime
+
+import jsonschema
+
+repo_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../'))
+sys.path.append(repo_path)
+
+from api import config, validators
```

Comment:
Do you mean `import api.config`?  When importing a package, modules / sub-packages are not imported and attached unless you specify that in `__init__.py`